frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/api/IGraphElement.java

```diff
@@ -0,0 +1,4 @@
+
+package com.virtuslab.gitmachete.frontend.graph.api;
```

Comment:
Actually, maybe the package should be called `elements` not `api`? AFAICS there are no references to this package from `frontendUiTable`, which suggests it isn't an "API" anyway

Reply:
Or maybe the other way round... consider splitting `frontendGraphImpl` and `frontendGraphApi` Gradle subprojects (you might want to jump onto PR #219 first), this should make it pretty clear what's exposed to the UI and what's not